worker/build.rs

```diff
@@ -16,6 +16,35 @@ fn main() {
     };
 
     let out_dir = env::var("OUT_DIR").unwrap();
+
+    // Compile Rust flatbuffers
+    let flatbuffers_declarations = planus_translation::translate_files(
+        &fs::read_dir("fbs")
+            .expect("Failed to read `fbs` directory")
+            .filter_map(|maybe_entry| {
+                maybe_entry
+                    .map(|entry| {
+                        let path = entry.path();
+                        if path.extension() == Some("fbs".as_ref()) {
+                            Some(path)
+                        } else {
+                            None
+                        }
+                    })
+                    .transpose()
+            })
+            .collect::<Result<Vec<_>, _>>()
+            .expect("Failed to collect flatbuffers files"),
+    )
+    .expect("Failed to translate flatbuffers files");
+
+    fs::write(
+        format!("{out_dir}/fbs.rs"),
+        planus_codegen::generate_rust(&flatbuffers_declarations)
```

Comment:
That's all? No need to tricks in generated fbs.ts file to make clippy happy and so on?

Reply:
I was also a bit surprised given the fact that we do literal `include!()`, but apparently Rust tracks where source code comes from and from path it is clear that file is auto-generated and it ignores all warnings from there :man_shrugging:

I'm not sure why that happens and lazy to search, but it works out for well for us here :slightly_smiling_face: